server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/BallotManifestUpload.java

```diff
@@ -93,7 +95,7 @@ private UploadedFile attemptFilePersistence(final File the_file,
                                               final String the_hash) {
     UploadedFile result = null;
     
-    if (Persistence.isEnabled()) {
+    if (Persistence.hasDB()) {
```

Comment:
This appears in several methods. It would be nice if we could factor it out to a higher level. Failing that, even using an early return would be nice, in that it would reduce the level of conditional nesting.